modules/imgproc/src/smooth.cpp

```diff
@@ -44,6 +44,9 @@
 #include "precomp.hpp"
 #include "opencl_kernels_imgproc.hpp"
 
+#ifdef HAVE_OPENVX
+#include "openvx_bindings.hpp"
```

Comment:
I suggest avoiding usage of these wrappers in OpenCV. Maybe we should wait for wrappers made by @apavlenko and use them.